bodhi/server/config.py

```diff
@@ -443,6 +443,9 @@ class BodhiConfig(dict):
         'waiverdb.access_token': {
             'value': None,
             'validator': _validate_none_or(six.text_type)},
+        'koji_build_web_url': {
```

Comment:
Let's name this more simply as ```koji_web_url``` - it's redundant to include "build" since Koji is a build system.